history-of-quakers/modernized/21-chapter-19.adoc

```diff
@@ -919,7 +956,7 @@ which he promised he would.
 But he continuing deficient in the case,
 Furly at last published the said letter in print,
 but without mentioning the name of him to whom it was written, only his character,
-Cuidam legato.^
+__Cuidam legato.__^
```

Comment:
I can't comment on the line I want to, but line 942 above, there's a book title in a footnote that got missed. "Prastantium..."

Reply:
Oh, I was going to the italics in the footnotes at the end in one big PR, if that's still okay. I forgot to keep doing them along the way at some point early on. :)

Reply:
Oh yes! I totally forgot this was in a footnote, never mind. Sorry!